Header header
int32 stampID
int32 sequenceID
int8 IsValid
float64 x
float64 y
float64 z
float64 vx
float64 vy
float64 vz
float64 timestamp
